Fixes for tutorial on how to write tutorials (#273)
ros-planning/moveit2_tutorials: doc/how_to_contribute/how_to_write_tutorials.rst

How to Write a MoveIt Tutorial
==============================

This guide explains how to write tutorials for the MoveIt documentation.
Tutorials are one of the most useful contributions you can make because they are the first thing many new users see.
This guide is intended for any contributor who wants to submit a new tutorial.

> If you are looking for how-to guides for using MoveIt, you can find them :doc:`here </doc/how_to_guides/how_to_guides>`.

Reviewer: nit
```suggestion
If you are looking for a how-to guide for using MoveIt, you can find one :doc:`here </doc/how_to_guides/how_to_guides>`.
```

------------
- Ubuntu 20.04
- ROS 2 Galactic
- MoveIt 2

Steps
-----

> 1. Fork the `MoveIt 2 Tutorials repository <https://github.com/ros-planning/moveit2_tutorials.git>`_ and start a new branch with an intuitive name (e.g., ``jack/how-to-tutorials``)

Reviewer 2: ```suggestion
1. Fork the `MoveIt 2 Tutorials repository <https://github.com/ros-planning/moveit2_tutorials.git>`_ and start a new branch with an intuitive name (e.g., ``jack/how-to-tutorials``).
```

> #. Create a new file in the ``doc/tutorials`` directory with a ``.rst`` extension. The title be a concise description (e.g., "MoveIt Quickstart in RViz").

Reviewer 3: ```suggestion
#. Create a new file in the ``doc/tutorials`` directory with a ``.rst`` extension. The title should be a concise description (e.g., "MoveIt Quickstart in RViz").
```

Reviewer 2: Good catch

> 

Reviewer 4: I think this should be "Lay out" instead of "Layout".

  Next Step
  ---------

  <Link to the next tutorial here.>

Further Reading
---------------
- :doc:`/doc/how_to_contribute/how_to_contribute_to_site`

> - :doc:`MoveIt Concepts </doc/concepts/tutorials>`

Reviewer 3: ```suggestion
- :doc:`MoveIt Concepts: Tutorials </doc/concepts/tutorials>`
```
